protokt-runtime/api/protokt-runtime.api

```diff
@@ -595,7 +595,7 @@ public abstract interface class protokt/v1/KtMessageDeserializer {
 	public abstract fun readSInt32 ()I
 	public abstract fun readSInt64 ()J
 	public abstract fun readString ()Ljava/lang/String;
-	public abstract fun readTag ()I
+	public abstract fun readTag-pVg5ArA ()I
```

Comment:
what's this?

Reply:
UInt nonsense.

Reply:
is this ABI going to be stable?

Reply:
I don't know. We already do this below - can we merge and revisit pre-1.0?

Reply:
I don't know. We already do this below - can we merge and revisit pre-1.0?